src/middlewared/middlewared/plugins/pool_/dataset_encryption_info.py

```diff
@@ -228,8 +228,7 @@ def path_in_locked_datasets(self, tls, path):
         # WARNING: _EXTREMELY_ hot code path. Do not add more
         # things here unless you fully understand the side-effects.
         if path.startswith('/dev/zvol/'):
-            # 10 comes from len("/dev/zvol/")
-            path = path[10:].replace('+', ' ')
+            path = path.removeprefix('/dev/zvol/').replace('+', ' ')
```

Comment:
What's the purpose of this change? It seems superfluous.

Reply:
Using `path[10:]` in one branch and `removeprefix` in the other made me think they're doing conceptually different things (they're not).

Reply:
Should I optimize the `/mnt/` also? It's a more frequently used branch.